src/util/rust_util.rs

```diff
@@ -0,0 +1,8 @@
+/// Const funciton for min value of two usize numbers.
+pub const fn min_of_usize(a: usize, b: usize) -> usize {
```

Comment:
Is this function used anywhere?

Reply:
It is in the commented out code for gencopy. I can remove it for now.